Makefile

```diff
@@ -301,6 +301,14 @@ test:
 	cd awxkit && $(VENV_BASE)/awx/bin/tox -re py3
 	awx-manage check_migrations --dry-run --check  -n 'missing_migration_file'
 
+github_ci_setup:
```

Comment:
Can we add a comment above the targets so `make help` prints a message for these new targets?